pkgs/swiftgen/swift2objc/lib/src/parser/parsers/declaration_parsers/parse_method_declaration.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:swift2objc/src/parser/parsers/parse_declarations_map.dart';
+
+import '../../../ast/_core/shared/parameter.dart';
+import '../../../ast/_core/shared/referred_type.dart';
+import '../../../ast/declarations/compounds/class_declaration.dart';
+import '../../_core/utils.dart';
+
+ClassMethodDeclaration parseMethodDeclaration(
+  JsonMap methodSymbolJson,
+  ParsedSymbolsMap parsedSymbolsMap,
+) {
+  return ClassMethodDeclaration(
+    id: parseSymbolId(methodSymbolJson),
+    name: parseSymbolName(methodSymbolJson),
+    returnType: _parseMehodReturnType(methodSymbolJson, parsedSymbolsMap),
```

Comment:
nit: Mehod/Method
Also on the param parser function

Reply:
fixed